Solutions/Azure Key Vault/Analytic Rules/NRT_KeyVaultSensitiveOperations.yaml

```diff
@@ -17,9 +17,9 @@ query: |
   let SensitiveOperationList = dynamic(
   ["VaultDelete", "KeyDelete", "SecretDelete", "SecretPurge", "KeyPurge", "SecretBackup", "KeyBackup"]);
   AzureDiagnostics
-  | extend ResultType = columnifexists("ResultType", "NoResultType")
-  | extend requestUri_s = columnifexists("requestUri_s", "None"), identity_claim_http_schemas_microsoft_com_identity_claims_objectidentifier_g = columnifexists("identity_claim_http_schemas_microsoft_com_identity_claims_objectidentifier_g", "None")
-  | extend id_s = columnifexists("id_s", "None"), CallerIPAddress = columnifexists("CallerIPAddress", "None"), clientInfo_s = columnifexists("clientInfo_s", "None")
+  | extend ResultType = column_ifexists("ResultType", "NoResultType")
```

Comment:
same changes suggested for the KeyVaultSensitiveOperations file.

Reply:
Please confirm, which changes do I need to do Here.